src/ui/components/results/directanswercomponent.js

```diff
@@ -181,4 +183,11 @@ export default class DirectAnswerComponent extends Component {
 
     this.analyticsReporter.report(event);
   }
+
+  addChild (data, type, opts) {
+    if (type === this._config.defaultCard) {
+      return super.addChild(this.getState(), type, opts);
```

Comment:
This is putting this component's (the parent's) config in the child component's `config.data._config`. I don't think we want that, since the child's `container` is different, and other config could be as well. A child could get the parent's config through `config._parentOpts`, and/or we could make the interface here super explicit, what do you think?

Reply:
Sure we can be a bit cleaner about what gets passed through

Reply:
Do we want to include some of the _parentOps in opts? For instance, would we want to include the various selector config options from the parent as top-level config for the child?

Reply:
I think it makes sense to include all of these from the README, what do you think @oshi97?

  // Optional, the selector for the form used for submitting the feedback
  formEl: '.js-directAnswer-feedback-form',
  // Optional, the selector to bind ui interaction to for reporting
  thumbsUpSelector: '.js-directAnswer-thumbUp',
  // Optional, the selector to bind ui interaction to for reporting
  thumbsDownSelector: '.js-directAnswer-thumbDown',
  // Optional, the display text for the View Details click to action link
  viewDetailsText: 'View Details',
  // Optional, the screen reader text for positive feedback on the answer
   positiveFeedbackSrText: 'This answered my question',
  // Optional, the screen reader text for negative feedback on the answer
   negativeFeedbackSrText: 'This did not answer my question',
  // Optional, the footer text to display on submission of feedback
   footerTextOnSubmission: 'Thank you for your feedback!'